website/docs/table-design/data-types.md

```diff
@@ -23,25 +23,27 @@ sidebar_position: 10
+| TIMESTAMP_LTZ(p)                                                         | A timestamp WITH time zone `TIMESTAMP WITH TIME ZONE` where p is the number of digits of fractional seconds (=precision). p must have a value between 0 and 9 (both inclusive). <br/>An instance consists of `year-month-day hour:minute:second[.fractional]` with up to nanosecond precision and values ranging from 0000-01-01 00:00:00.000000000 to 9999-12-31 23:59:59.999999999. <br/> Compared to the SQL standard, leap seconds (23:59:60 and 23:59:61) are not supported as the semantics are closer to java. time. LocalDateTime |
+| BINARY(n)                                                                | A fixed-length binary string (=a sequence of bytes) where n is the number of bytes. n must have a value between 1 and Integer.MAX_VALUE (both inclusive).                                                                                                                                                                                                                                                                                                                                                                                 |
+| BYTES                                                                    | A variable-length binary string (=a sequence of bytes).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| ARRAY&lt;t&gt;                                                           | An array of elements with same subtype. <br/>Compared to the SQL standard, the maximum cardinality of an array cannot be specified but is fixed at Integer.MAX_VALUE. Also, any valid type is supported as a subtype.                                                                                                                                                                                                                                                                                                                     |
```

Comment:
But `ARRAY`, `MAP`, `ROW` is still not supported in Fluss.

Reply:
Thanks, I will close this pr.

Reply:
But BINARY is supported, which is missing in this doc. I think you can still open this pr, but only add `BINARY`

Reply:
done.